src/genai/tune/tune_service.py

```diff
@@ -65,9 +66,9 @@ def create(
                 name=name,
                 parameters=to_model_optional(parameters, TuneParameters),
                 task_id=task_id,
-                training_file_ids=training_file_ids,
+                training_file_ids=list(training_file_ids),
```

Comment:
Incorrect.

Reply:
This converts `Sequence[str]` => `list[str]`, which is accepted by `_TuneCreateRequest` so  why is it incorrect?

Reply:
Because user can pass `None`